src/app/components/ReadTime/index.tsx

```diff
@@ -0,0 +1,22 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import styles from './index.styles';
+
+type ReadTimeProps = {
+  readTimeValue: number | undefined;
+  className?: string;
+};
+
+const ReadTime =({ readTimeValue, className }: ReadTimeProps) => {
+    if (readTimeValue === undefined) return false;
+
+  return (
+    <div className={className} css={styles.prototype}>
+      <p>
+        read time: { (readTimeValue + " minutes") || "no read time supplied"}
```

Comment:
"no read time supplied" will never be displayed because of the check on line 11. So let's remove it.

Also, let's not use string concatenation `+`
```suggestion
        Estimated Read Time: {readTime} minutes
```

Reply:
In a future iteration, we would need translations

Reply:
As part of the future translations work, we might need to also consider singular vs plural - 1 minute vs 2 minutes

Reply:
I've made a start on translations for "Estimated Read Time", would we also need translations for "minutes"?

Reply:
Let's get a placeholder translation for Estimated Read Time into the Service Config - and perhaps use Google Translate temporarily for a service of your choosing 😸 

RE translation for minutes - we should be able to use the `moment` library - see https://latest--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/translations-timestamps--example for some examples (change the service in the header to get it in different languages). 

I suggest that we do this in a future PR so that we can move quickly on this PR & get it merged - and I can help you work out how we might do this.

Reply:
cool, we can have a look at timestamps later, I've added a translation for Mundo to test for now _"Tiempo estimado de lectura"_!